mfjcs-rest/src/main/java/org/mfjcs/api/ItemMetadata.java

```diff
@@ -0,0 +1,11 @@
+package org.mfjcs.api;
+
+public interface ItemMetadata {
+
+	String getId();
```

Comment:
regarding earlier remark on slack about namespacing id's
should we introduce a tuple-string for id, or decide on a separator? 
or ignore it on this level, and just see it as part of the id being always 'universal'